cosmic-settings/src/pages/power/mod.rs

```diff
@@ -38,18 +44,43 @@ impl page::Page<crate::pages::Message> for Page {
         _page: cosmic_settings_page::Entity,
         _sender: tokio::sync::mpsc::Sender<crate::pages::Message>,
     ) -> cosmic::Command<crate::pages::Message> {
-        cosmic::command::future(async move {
-            let battery = Battery::update_battery().await;
-            Message::UpdateBattery(battery)
-        })
-        .map(crate::pages::Message::Power)
+        let futures: Vec<Command<Message>> = vec![
+            cosmic::command::future(async move {
+                let battery = Battery::update_battery().await;
+                Message::UpdateBattery(battery)
+            }),
+            cosmic::command::future(async move {
+                let devices = ConnectedDevice::update_connected_devices().await;
+                // let devices = vec![
+                //     ConnectedDevice {
+                //         model: "Test device 1".to_owned(),
+                //         device_icon: "display-symbolic",
+                //         battery: Battery::default(),
+                //     },
+                //     ConnectedDevice {
+                //         model: "Test device 2".to_owned(),
+                //         device_icon: "laptop-symbolic",
+                //         battery: Battery::default()
+                //     },
+                //     ConnectedDevice {
+                //         model: "Test device 3".to_owned(),
+                //         device_icon: "network-wired-symbolic",
+                //         battery: Battery::default()
+                //     },
+                // ];
```

Comment:
I will remove this before merging, but thought I will keep it in during review to help with testing!